data/network/src/main/java/io/filmtime/data/network/NetworkModule.kt

```diff
@@ -24,16 +27,38 @@ object NetworkModule {
     }
   }
 
+  @Provides
+  @Singleton
+  fun providesInterceptor(): Map<Int, @JvmSuppressWildcards Interceptor> {
+    return mapOf(
+      1 to ApiKeyInterceptor()
```

Comment:
As @hadi-norouzi mentioned please use annotation class e.g: ForTmdb

```suggestion
  @ForTmdb
  @Provides
  @Singleton
  fun providesTmdbInterceptor(): Interceptor {
    return ...
```